config/routes.rb

```diff
@@ -227,6 +231,9 @@
     post "/notification_subscriptions/:notifiable_type/:notifiable_id", to: "notification_subscriptions#upsert"
     patch "/onboarding_update", to: "users#onboarding_update"
     patch "/onboarding_checkbox_update", to: "users#onboarding_checkbox_update"
+    # rubocop:disable Layout/LineLength
```

Comment:
I'd break down the route definition instead of ignoring the line length, which ends up taking less vertical space in total:

```ruby
patch "/onboarding_notifications_checkbox_update", 
      to: "users/notification_settings#onboarding_notifications_checkbox_update"
```